README.md

```diff
@@ -1,7 +1,73 @@
 # Jitterbug
 A Laravel / MySQL database management application to support large-scale description, digitization, preservation and access of archival audiovisual recordings in the Wilson Special Collections Library of UNC-Chapel Hill. Funded by an Andrew W. Mellon Foundation grant, "Extending the Reach of Southern Audiovisual Sources."
 
-## Requirements
+
+## Initial setup
+1. If you are not a member of the Jitterbug LDAP group, submit a ticket and request access. You must be a member of the group in order to log into Jitterbug.
+2. Get a fresh Jitterbug MySQL production database dump (`$jitterbug-db-dump` refers to the name of this file) from a UNC library sysadmin.
+3. Because the dump will contain views, you will need to remove the SQL security definers in the dump file. Using sed (on Mac OS X):
+```bash
+$ sed -i '' 's/DEFINER=[^*]*\*/\*/g' $jitterbug-db-dump
+```
+---
+## Vagrant Installation
+If you would like to use a pre-configured Vagrant box, Laravel Homstead is available for use.
+
+### Installation steps
+1. Clone the jitterbug repository.
+```bash
+$ git clone https://github.com/UNC-Libraries/jitterbug.git
+```
+2. `$jitterbug-home` is the directory where you've cloned the jitterbug project. Navigate to it.
+```bash
+$ cd $jitterbug-home
+```
+3. Install composer by following directions at https://getcomposer.org/download/
+
+4. Set up the MySQL Connector.
+    1. Get the MySQL connector [file](http://www.mirrorservice.org/sites/ftp.mysql.com/Downloads/Connector-J/mysql-connector-java-8.0.15.zip) and unzip it.
```

Comment:
I wonder if this step could be added to `after.sh`

Reply:
If the server has access to `wget` and `unzip` i probably could? Emily suggested using `wget` and i started writing up instructions for it, but then wasn't sure if it was expected that everyone had those commands in their terminal.